Foundation/Stream.swift

```diff
@@ -116,15 +114,35 @@ open class Stream: NSObject {
 open class InputStream: Stream {
 
     internal let _stream: CFReadStream!
-
+    internal weak var _delegate: StreamDelegate?
+    
+    open override var delegate: StreamDelegate? {
+        get {
+            return _delegate
+        }
+        set {
+            if let newValue = newValue {
+                let context = CFStreamClientContext(version: 0, info: &self, retain: nil, release: nil, copyDescription: nil))
+                if CFReadStreamSetClient(_stream, .max, InputStream.clientCallback, context) {
+                    delegate = newValue
```

Comment:
Should this be `_delegate` ? (and the same on line 224)

Reply:
Fixed thanks